File: flow_camp_app/lib/models/interest.dart
class Interest {
  final int id;
  final String name;
  final String category;
  final bool confirmed;

  Interest({
    required this.id,
    required this.name,
    required this.category,
    required this.confirmed,
  });

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'category': category,
      'confirmed': confirmed,
    };
  }

  factory Interest.fromJson(Map<String, dynamic> json) {
    return Interest(
      id:json['id'],
      name: json['name'],
      category: json['category'],
      confirmed: json['confirmed'],
    );
  }
}

File: flow_camp_server/models/interest.js
const Sequelize = require('sequelize');

class Interest extends Sequelize.Model {
    static initiate(sequelize) {
        Interest.init({
            name:{
                type: Sequelize.STRING(100),
                allowNull: true,
            },
            category: {
                type: Sequelize.STRING(100),
                allowNull: true,
            },
            confirmed:{
                type: Sequelize.BOOLEAN,
                allowNull: false,
                defaultValue: false,
            }
        }
            , {
                sequelize,
                timestamps: false,
                modelName: 'Interest',
                tableName: 'interest',
                paranoid: false,
                charset: 'utf8mb4',
                collate: 'utf8mb4_general_ci',

            });
    }
    static associate(db) {
        db.Interest.belongsToMany(db.User,{through:'UserInterest'});
    }
};
module.exports = Interest;